### PR TITLE
Update in the About ATLAS page in the CERN Open Data portal

### PR DESCRIPTION
**Pull request by artfisica**
Hi!

This request is an update in the **'About ATLAS'** page in the CERN Open Data portal.
Taking inspiration from the CMS pages and adding all the latest domains to the ATLAS Open Data resources.

Thanks!
Arturo

### CODE REVIEW

**Comment by artfisica** [comment]
Dear All,

Sorry again if I made this request in the wrong way. 
Since it is still unassigned, maybe I need to re-do it more properly?

Thanks as usual,
Arturo

File: cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md [hunk 1]
```diff
@@ -1,7 +1,54 @@
-ATLAS (A Toroidal LHC ApparatuS) is one of the two general-purpose detectors at the LHC, the other being CMS. Both detectors are geared towards covering a large range of physics produced in proton collisions such as the search for the Higgs Boson or Vector Boson Scattering.
+**[ATLAS](https://atlas.cern)** (A Toroidal LHC ApparatuS) is one of the four major experiments at the [Large Hadron Collider](http://home.web.cern.ch/topics/large-hadron-collider) (LHC) at [CERN](https://home.cern). It is a general-purpose particle physics experiment run by an international collaboration and, together with [ATLAS](https://atlas.cern), is designed to exploit the full discovery potential and the vast range of physics opportunities that the LHC provides.
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md]: Need to change the text to `[CMS](https://cms.cern)`.

File: cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md [hunk 2]
```diff
@@ -1,7 +1,54 @@
...
 
-## About ATLAS Data
+[ATLAS](https://atlas.cern)' scientific exploration uses precision measurement to push the frontiers of knowledge by seeking answers to fundamental questions such as: What are the basic building blocks of matter? What are the fundamental forces of nature? Could there be a greater underlying symmetry to our universe?
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md]: ATLAS' &rarr; ATLAS's

File: cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md [hunk 3]
```diff
@@ -1,7 +1,54 @@
...
\ No newline at end of file
+For more about the ATLAS Collaboration, the experiment, the physics and its members, please visit the [Official Site](https://atlas.cern/discover/experiment).
+
+
+## About [ATLAS Open Data](http://opendata.atlas.cern)
+
+**The ATLAS Collaboration's current approach on the release of datasets is intended for Education, Training and Outreach activities around the World**.
+In order to fulfil that objective, the **[ATLAS Open Data project](http://opendata.atlas.cern) was created**.
+
+[ATLAS Open Data](http://opendata.atlas.cern) project aims to provide data and tools to high school, masters and undergraduate students, to help educate them in physics analysis techniques used in experimental particle physics. Sharing data collected by the [ATLAS](https://atlas.cern) experiment aims to generate excitement and enthusiasm for fundamental research, inspiring physicists of the future.
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md]: high school &rarr; high-school

File: cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md [hunk 4]
```diff
@@ -1,7 +1,54 @@
...
+
+[International Masterclasses](http://physicsmasterclasses.org/) introduce particle physics to high school students and have been running very successfully for the last 20 years.
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md]: high school &rarr; high-school

File: cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md [hunk 5]
```diff
@@ -1,7 +1,54 @@
...
+ATLAS data [were first used in 2011](https://www.interactions.org/press-release/high-school-students-analyse-first-time-real-lhc-data), and 8 TeV ATLAS Open Data in 2015.
+Inspired by the success of International Masterclasses, a new [ATLAS Open Data](http://opendata.atlas.cern) initiative [was launched in 2016](http://atlas.cern/updates/atlas-news/explore-lhc-data-new-atlas-educational-platform).
+
+ The target audience is physics undergraduate and masters students, but may also include advanced high school students or indeed anyone with some basic understanding of particle physics and coding experience. A comprehensive **educational platform was developed featuring a new 8 TeV dataset and a set of educational tools at a more advanced level. The objectives of this initiative are to provide**:
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md]: high school &rarr; high-school

File: cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md (hunk 1 above)
Comment (artfisica) [reply cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md]: Thank you for spotting this.
Done.

File: cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md (hunk 2 above)
Comment (artfisica) [reply cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md]: Done.

File: cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md (hunk 5 above)
Comment (artfisica) [reply cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md]: Thanks

**Comment by artfisica** [comment]
changes done.

**Comment by artfisica** [comment]
Hi!
I would like to ask if there is something else to be done from our side?
Thanks
Arturo

**Comment by tiborsimko** [comment]
Thanks, I squashed your branch and created a new PR #2409 in your name, so I'm closing this one.

P.S. BTW please note that we usually don't use merge commits in feature branches to keep nice commit log history. We usually `git rebase master -i` and repush the feature branch for final merging, see https://github.com/cernopendata/opendata.cern.ch/blob/master/DEVELOPING.rst#updating-pull-requests

**Pull request closed without merging**